README.md

```diff
@@ -169,31 +172,43 @@ For faster production-tier searches, Qdrant also provides a gRPC interface. You
 
 ### Filtering and Payload
 
-Qdrant enables JSON payloads to be associated with vectors, providing both storage and filtering based on payload values. It supports various combinations of `should`, `must`, and `must_not` conditions, ensuring retrieval of all relevant vectors unlike `ElasticSearch` post-filtering.
+Qdrant can attach any JSON payloads to vectors, allowing for both the storage and filtering of data based on the values in these payloads.
+Payload supports a wide range of data types and query conditions, including keyword matching, full-text filtering, numerical ranges, geo-locations, and more.
 
-### Rich Data Types
+Filtering conditions can be combined in various ways, including `should`, `must`, and `must_not` clauses,
+ensuring that you can implement any desired business logic on top of similarity matching.
 
-The vector payload accommodates diverse data types and query conditions, including string matching, numerical ranges, geo-locations, and more. These filtering conditions empower you to create custom business logic on top of similarity matching.
 
-### Query Planning and Payload Indexes
+### Hybrid Search with Sparse Vectors
 
-The _query planner_ leverages stored payload information to optimize query execution. For instance, smaller search spaces limited by filters might benefit from full brute force over an index.
+To address the limitations of vector embeddings when searching for specific keywords,
+Qdrant introduces support for sparse vectors in addition to the regular dense ones.
 
-### SIMD Hardware Acceleration
+Sparse vectors can be viewed as an extension of BM25 or TF-IDF searches.
+However, they also harness the capabilities of transformer-based neural networks to weigh individual tokens effectively.
```

Comment:
Want to highlight that this extension/generalisation is a good thing, and not a limitation with "However"

```suggestion
Sparse vectors can be viewed as an generalisation of BM25 or TF-IDF ranking. They enable you to harness the capabilities of transformer-based neural networks to weigh individual tokens effectively.
```